teams/teams-ps/teams/New-CsOnlineLisLocation.md

```diff
@@ -17,18 +17,8 @@ Use the New-CsOnlineLisLocation cmdlet to create a new emergency dispatch locati
 
 ## SYNTAX
 
-### ExistingCivicAddress (Default)
 ```
-New-CsOnlineLisLocation -Location <string> -CivicAddressId <guid> [-CityAlias <string>] [-CompanyName <string>] [-CompanyTaxId <string>] [-Confidence <string>]
-[-Elin <string>] [-Force] [-HouseNumberSuffix <string>] [-Latitude <string>] [-Longitude <string>] [-WhatIf] [-Confirm] [<CommonParameters>]
-```
-
-### CreateCivicAddress
-```
-New-CsOnlineLisLocation -Location <string> -CountryOrRegion <string> [-CityAlias <string>] [-CompanyName <string>] [-CompanyTaxId <string>] [-Confidence <string>]
-[-Elin <string>] [-Force] [-HouseNumberSuffix <string>] [-Latitude <string>] [-Longitude <string>] [-City <string>] [-Description <string>] [-HouseNumber <string>]
-[-PostalCode <string>] [-PostDirectional <string>] [-PreDirectional <string>] [-StateOrProvince <string>] [-StreetName <string>] [-StreetSuffix <string>]
-[-WhatIf] [-Confirm] [<CommonParameters>]
+New-CsOnlineLisLocation -Location <string> -CivicAddressId <guid> [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
I think we also need to add -Elin which is missing

Reply:
done